lib/logstash/plugin_mixins/elasticsearch/common.rb

```diff
@@ -268,7 +268,7 @@ def submit(actions)
           @document_level_metrics.increment(:non_retryable_failures)
           @logger.warn "Failed action", status: status, action: action, response: response if log_failure_type?(error)
           next
-        elsif DOC_DLQ_CODES.include?(status)
+        elsif DOC_DLQ_CODES.include?(status) || dlq_custom_codes.include?(status)
```

Comment:
I would prefer using a `Set` (loaded once at plugin registration) that contained the union of these two, loaded into an instance variable, instead of adding a per-event overhead.

Reply:
Fixed by ec57f24c27fdb595b9c78e13a2c67ce754fd7926